02-04-2015/coins/coins-chintan.txt

```diff
@@ -2,12 +2,16 @@ class Coins
 	COINS = [10,5,2,1,0.50,0.25]
 	def self.count(amount)
 		coin_count = 0
```

Comment:
yes, we can simply pass '0' value into recursive method instead of passing variable.